### PR TITLE
Update Arm delegate tutorial for MobileNetV2 usages

### PR DESCRIPTION
**Pull request by Jerry-Ge**
Add the related setup documentations for running
the Delegated Quantized MobileNetV2 example

Change-Id: Ied8298938fcb37cfa95bd044439c305380f0c1eb

### CODE REVIEW

File: docs/source/executorch-arm-delegate-tutorial.md [hunk 1]
```diff
@@ -316,7 +327,43 @@ python3 -m examples.arm.aot_arm_compiler --model_name="add" --delegate
 # should produce ./add_arm_delegate.pte
 ```
 
-At the end of this, we should have two different `.pte` files. First one with the [SoftmaxModule](#softmaxmodule), without any backend delegates. And the second one with the [AddModule](#addmodule), and with Arm Ethos-U backend delegate enabled. Now let's try to run these `.pte` files on a Corstone-300 platform in a bare-metal environment.
+### Delegated Quantized Workflow
+Before generating the `.pte` file for delegated quantized networks like MobileNetV2, we need to build the `quantized_ops_aot_lib`
```
Comment (digantdesai) [inline docs/source/executorch-arm-delegate-tutorial.md:331]: Nit: Is this covered by some other docs? If not does it make sense to create another page and link it from here? Rationale is if other use cases need to do this as well.

**Review by digantdesai — APPROVED** [review]

File: docs/source/executorch-arm-delegate-tutorial.md (hunk 1 above)
Comment (Jerry-Ge) [reply docs/source/executorch-arm-delegate-tutorial.md:331]: > Is this covered by some other docs?
No from my understanding. That makes sense. I will refactor this part once we got more use cases. I will leave this as this right now.

**Comment by digantdesai** [comment]
Also you might want to rebase on top of the #3847 to make sure the CI is clean

**Comment by Jerry-Ge** [comment]
> Also you might want to rebase on top of the #3847 to make sure the CI is clean

just rebased! thanks for the note.

**Pull request closed without merging**